packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -331,6 +331,11 @@ location /package/ {
     include includes/disable-request-response-buffering.conf;
     include includes/http-11.conf;
 
+    # Package installation can require large downloads, increasing the chance
+    # for intermittent network connectivity to cause a problem.  Increase
+    # timeouts to make package download more robust.
```

Comment:
Let's note here that Cosmos buffers the entire "package file" before passing it on to nginx, and that this is an architectural limitation that might be addressed in the future

Reply:
Let's then also mention DCOS_OSS-3639 in the comment.